packages/contracts-bedrock/scripts/slither.sh

```diff
@@ -2,12 +2,31 @@
 
 rm -rf artifacts forge-artifacts
 
+set -e
+
+if ! command -v slither &> /dev/null
+then
+    echo "Slither could not be found. Please install slither by running:"
+    echo "pip3 install slither-analyzer"
+    exit
+fi
+
+if [ -e "slither-report.json" ]; then
+    rm "slither-report.json"
+    echo "slither-report.json deleted."
+fi
+
+dir=$(dirname "$0")
+
 # See slither.config.json for slither settings
 if [[ -z "$TRIAGE_MODE" ]]; then
-  echo "Building contracts"
-  forge build --build-info --force
-  echo "Running slither"
-  slither --ignore-compile .
+  echo "Running slither in normal mode"
+  slither . &> /dev/null || true
```

Comment:
why or true?

Reply:
There's currently around 60 report items. If one is generated this command exits unsuccessfully. This pr has not triaged all these errors. If we want to triage I highly suggest stacking ontop and giving this task to the security team.